src/TextInput/index.js

```diff
@@ -50,6 +54,17 @@ const Suffix = ({ children }) => {
   return <View style={styles.suffix}>{suffix}</View>;
 };
 
+const ClearButton = ({ onPress, focused, value }) => {
+  if (focused && value) {
```

Comment:
I know this is what the design says, but I find it weird that I cannot reset the input field if it's not in focused state and there is a value in it. I need two touches to reset the input field. WDYT?
@tbergq thoughts?

Reply:
In the majority of the inputs, clear button appears when input is focused and value.length > 0.

Reply:
I would say stick to design, and rather create an issue with designers